bootstrap/ncpms_seed.py

```
@@ -0,0 +1,74 @@
+from _bootstrap_common import psql_query, info
+
+ROOT = Path(__file__).resolve().parent.parent
+NCPMS_JSON_PATH = ROOT / "tools" / "ncpms-api-crawler" / "json_raw" / "ncpms_data.json"
+
+async def run_ncpms_seed(db_conf: dict[str, str]):
+    info("NCPMS 캐시 테이블 적재 스크립트 실행")
+    if not NCPMS_JSON_PATH.exists():
+        info(f"NCPMS 데이터 파일 없음: {NCPMS_JSON_PATH} (의도적 스킵 가능)")
+        return
+
+    with open(NCPMS_JSON_PATH, "r", encoding="utf-8") as f:
```

Ruff (0.15.10)

[warning] Async functions should not open files with blocking methods like `open`

(ASYNC230)